CHANGELOG.md

```diff
@@ -20,11 +20,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
  - Queryables landing page and collection links when the Filter Extension is enabled [#267](https://github.com/stac-utils/stac-fastapi-elasticsearch-opensearch/pull/267)
+ - Added support for route depndencies configuration through the STAC_FASTAPI_ROUTE_DEPENDENCIES environment variable, directly or via json file. Allows for fastapi's inbuilt OAuth2 flows to be used as dependencies. Custom dependencies can also be written, see Basic Auth for an example. [#251](https://github.com/stac-utils/stac-fastapi-elasticsearch-opensearch/pull/251)
```

Comment:
These changes have to be moved to the Unreleased section. They can't be in the v3.0.0a2 section because v3.0.0a2 has already been released.

Reply:
Good spot sorry should have checked the merge a bit more thoroughly should be fixed now 👍

Reply:
Thanks - something easy to miss